client/src/components/calender/Calender.tsx

```diff
@@ -41,26 +45,65 @@ const Calendar = (): ReactElement => {
     modifiedArg.el.style.cursor = 'pointer';
     modifiedArg.el.style.backgroundColor = '#2CB674';
   };
+
+  const AddNewEvent = async (newEvent: newEventInterface) => {
+    await setDate(
```

Comment:
serDate function does not return a `promise`, so there is no need to use `await`